src/release.js

```diff
@@ -72,7 +71,7 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
     // Initialise the Adaptee and give it a copy of the release params and loaded data
     if (subjectType === 'extension') {
         adaptee = extensionApi(params, data);
-    } else if (subjectType === 'package') {
+    } else if (subjectType === 'package' || subjectType === 'repo') {
```

Comment:
The `packageApi` does require a package.json file, which limits the kinds of repos it can release... what's the intended difference between `npmRelease` and `ghRelease` commands?

Reply:
The `subjectType` is also used as an object key later, for extracting the name - that's why `taoRelease ghRelease` isn't working yet. I guess you plan for a more generic `repoApi`?

Reply:
My intent was to just reuse the package type without the dedicated function, but now I've the full implementation